packages/augur-ui/src/modules/auth/actions/logout.ts

```diff
@@ -17,8 +17,7 @@ export function logout() {
     if (localStorageRef && localStorageRef.removeItem) {
       localStorageRef.removeItem('airbitz.current_user');
       localStorageRef.removeItem('airbitz.users');
-      localStorageRef.removeItem('loggedInAccount');
-      localStorageRef.removeItem('loggedInAccountType');
+      localStorageRef.removeItem('loggedInUser');
```

Comment:
Should we be removing the pending safes here too, or do you need those hanging around?

Reply:
we should keep them around to restore safes properly for users with many safes.